docs/security/certificate-management.md

```diff
@@ -799,7 +300,7 @@ NOTE:
     Run `osg-ca-manage refreshCA` and `osg-ca-manage fetchCRL` to commit your changes immediately.
 ```
 
-#### Inspect Installed CA Certificates ####
+#### Inspect Installed CA Certificates
```

Comment:
h4 `List installed CA certificates`

Only h1 and h2 sections should be title case. Can you check to make sure all the other sections have proper capitalization?

Reply:
I have done the requessted changes @brianhlin can you please review?

Reply:
Thanks for noticing up @brianhlin can you merge now?